cves/kernel/CVE-2016-2545.yml

```diff
@@ -278,8 +298,14 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
+    The snd_timer_interrupt function, where the vulnerability resides, does not
```

Comment:
It is unlikely that snd_timer_interrupt is using any forms of IPC as it is a low-level timer interrupt, so it is very unlikely that it is related to any form of userspace IPC (i.e. sockets, pipes, etc.) as it deals with hardware directly.

Reply:
I take this as you agreeing to `false` and most of what I had?

I rewrote to align with your comment, 66647f367bf25a0840720cfeba3b4ce0e3035b8e, is that fine? Thanks.